modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/BaseSqlMultiStatementTest.java

```diff
@@ -107,6 +115,7 @@ static List<AsyncSqlCursor<InternalSqlRow>> fetchCursors(AsyncSqlCursor<Internal
             assertNotNull(cursor);
 
             cursors.add(cursor);
+            cursorsToClose.add(cursor);
```

Comment:
do we need to check "close" flag here ? 
+ do we need to append initial cursor i.e. cursor from line [109](https://github.com/apache/ignite-3/pull/4767/files#diff-09c4235d0030bbc86e6d0d24768feae47f2a08e26c7fb2e6fa194a11ec786d57R109) here ?

Reply:
Yes, I missed it, fixed, thanks